junit/README.md

```diff
@@ -0,0 +1,50 @@
+Cucumber JUnit 
+==============
+
+Use JUnit to execute cucumber scenarios.
+
+Add the `cucumber-junit` dependency to your pom.
+
+```xml
+<dependencies>
+  [...]
+    <dependency>
+        <groupId>io.cucumber</groupId>
+        <artifactId>cucumber-junit</artifactId>
+        <version>${cucumber.version}</version>
+        <scope>test</scope>
+    </dependency>
+  [...]
+</dependencies>
+```
+
+Create an empty class that uses the Cucumber JUnit runner.
+
+```java
+package com.example;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(plugin = "json:target/cucumber-report.json")
+public class RunCukesTest {
+}
+```
+
+This will execute all scenarios in the package of the runner. By default glue code is assumed to be in the same 
```

Comment:
[text edit suggestion] "This will execute all scenarios in the *same* package *as* the runner; by default glue code is assumed to be in the same package."

Reply:
[suggestion] Add an example of how to use the '@CucumberOptions'?

Reply:
Do we have some documentation about cucumber options to link here?

Reply:
Not atm. Fyi, it's just a suggestion. Not blocking at all.